semantic/src/themes/universe/elements/segment.variables

```diff
@@ -1,5 +1,7 @@
 @boxShadow: 0;
 
+@borderColor: @orbit;
```

Comment:
Making sure that the segments are using orbit as the border colour instead of default rgb. (Barely any difference, but this is perfection?)